src/fstab-generator/fstab-generator.c

```diff
@@ -692,7 +693,11 @@ static int parse_fstab(bool initrd) {
                                 continue;
                         }
 
-                        if (is_device_path(what)) {
+                        e = getenv_bool_secure("SYSTEMD_FORCE_FSTAB");
+                        if (e < 0 && e != -ENXIO)
+                                log_debug_errno(r, "Failed to parse $SYSTEMD_FORCE_FSTAB, ignoring: %m");
+
+                        if (e <= 0 && is_device_path(what)) {
                                 log_info("/sys/ is read-only (running in a container?), ignoring fstab device entry for %s.", what);
                                 continue;
                         }
```

Comment:
hmm, I think it would be better to just explicitly override the sysfs read-only check altogether. i.e. call the env var SYSTEMD_SYSFS_CHECK=0/1 or so.

Reply:
Done, I've renamed it and flipped what it does to follow the new name. I didn't update the description in `docs/ENVIRONMENT.md` much, so let me know if I need to adjust any of that